packages/2d/src/components/Curve.ts

```diff
@@ -49,10 +55,16 @@ export abstract class Curve extends Shape {
   @initial(0)
   @signal()
   public declare readonly endOffset: SimpleSignal<number, this>;
-
+  /**
+   * Specifies if or what arrow should be drawn at the start of the line.
+   *
+   * @remarks
+   * If `true`, a default arrow will be drawn. This can be used with the {@link arrowSize} signal to change the size of the arrow.
+   * If set to a {@link Node}, the node will be used as the arrow. This can be used to edraw a custom arrow. If a custom arrow is used, the {@link arrowSize} signal will be ignored.
```

Comment:
```suggestion
   * If set to a {@link Node}, the node will be used as the arrow. This can be used to draw a custom arrow. If a custom arrow is used, the {@link arrowSize} signal will be ignored.
```